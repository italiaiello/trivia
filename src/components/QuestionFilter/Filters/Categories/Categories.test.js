import React from 'react'
import { shallow } from 'enzyme'
import Categories from './Categories'

it('checks if it Categories.js renders properly', () => {

    expect(shallow(<Categories />)).toMatchSnapshot()
})