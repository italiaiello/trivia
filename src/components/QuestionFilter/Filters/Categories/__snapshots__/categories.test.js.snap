// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if it Categories.js renders properly 1`] = `ShallowWrapper {}`;
